package.json

```diff
@@ -162,7 +163,8 @@
     "dev:webui": "cross-env BABEL_ENV=ui babel-node tools/dev.server.js",
     "build:webui": "npm run pre:webpack && BABEL_ENV=ui webpack --config tools/webpack.prod.config.babel.js",
     "build:docker": "docker build -t verdaccio . --no-cache",
-    "build:docker:rpi": "docker build -f Dockerfile.rpi -t verdaccio:rpi ."
+    "build:docker:rpi": "docker build -f Dockerfile.rpi -t verdaccio:rpi .",
+    "postinstall": "opencollective postinstall"
```

Comment:
Honestly, I do not like the `"postinstall": "opencollective postinstall"` script. It's too invasive and out of our control. How do I ensure that won't fails and hit the user experience?

Just a quick look up: https://github.com/search?q=opencollective+postinstall&type=Issues&utf8=%E2%9C%93

Reply:
This is the output each time an user install `verdaccio`. 
```
> opencollective postinstall

                                                             
                              ......        ........         
                              ;33333r      ,;s22222r.        
                              .rGhhhGr   ,:;r2XXXXs.         
                               .rhhhhGr  .,,,:::::.          
                                .rhhhhGr  :siiii;            
                                 .rhhhhGr;9A&&AS.            
                                  .rhhhhGG&&&&S.             
                                   .rhhhhG&&&S.              
                                    .rhhhhG&S.               
                                     .rhhhG5.                
                                      .;;;;.                 
                                         

                        Thanks for installing verdaccio 🙏
                 Please consider donating to our open collective
                        to help us maintain this package.

                            Number of contributors: 98
                               Number of backers: 0
                               Annual budget: US$ 0
                              Current balance: US$ 0

         👉  Donate: https://opencollective.com/verdaccio/donate

````

Reply:
Yes, the default makes a network request everytime and shows an ASCII version of your logo. ASCII art is hit or miss. You can remove dependency altogether and use the native way like here: https://github.com/remy/nodemon/blob/master/package.json#L42 :-)

Reply:
You can also completely remove/ignore doing any post install request for the collective.

Reply:
> the default makes a network request everytime and shows an ASCII version of your logo.

Yeah, this might be a problem, many users work behind firewalls or offline completely. 

>https://github.com/remy/nodemon/blob/master/package.json#L42 :-)

Ok, this looks more affordable 👍  I'll ask core contributors.

> You can also completely remove/ignore doing any post install request for the collective.

I like the readme update, it looks pretty neat. But since the script will affect the user experience pretty likely I'll have to remove it.